pkg/types/entries.go

```diff
@@ -82,6 +83,12 @@ func CreateVersionedEntry(pe models.ProposedEntry) (EntryImpl, error) {
 		if !tf.(func() TypeImpl)().IsSupportedVersion(ei.APIVersion()) {
 			return nil, fmt.Errorf("entry kind '%v' does not support inserting entries of version '%v'", kind, ei.APIVersion())
 		}
+	} else {
+		return nil, fmt.Errorf("unknown kind '%v' specified", kind)
+	}
+
+	if ok, err := ei.Insertable(); !ok {
```

Comment:
To confirm, this should not be a breaking change to existing entries, correct? It looks like this is only called on create - https://github.com/sigstore/rekor/blob/3282e1ab3e9b8f773e8a44861a31a3df9adf01ce/pkg/api/entries.go#L178

Reply:
Correct, it should only affect all new entries put into the log